src/jit/gentree.cpp

```diff
@@ -14099,7 +14099,7 @@ GenTree* Compiler::gtFoldExprConst(GenTree* tree)
                     {
                         f1 = forceCastToFloat(d1);
                         f2 = forceCastToFloat(d2);
-                        d1 = f1 + f2;
+                        d1 = forceCastToFloat(f1 + f2);
```

Comment:
is the `non-x86 arch` comment above still valid? Now that x86 uses SSE/SSE2 (rather than the FPU stack), the comment should be applicable for all architectures. Right?

Reply:
Yes, I will update it. Thanks.